JustSaying.Models/Message.cs

```diff
@@ -8,11 +8,20 @@ protected Message()
         {
             TimeStamp = DateTime.UtcNow;
             Id = Guid.NewGuid();
+            CorrelationId = Id;
```

Comment:
We must also make best efforts to keep deserialization compatibility with previous versions, we might be alright here, but we should double check